### DISCUSSION TITLE
Please update your log levels to better match intent

### DISCUSSION THREAD

**Discussion by not1battu** [SixLabors/ImageSharp.Web#347] [Q&A] [answered]
The log levels defined here do not make sense: https://github.com/SixLabors/ImageSharp.Web/blob/main/src/ImageSharp.Web/Middleware/LoggerExtensions.cs#L34

`LogServedAction` should be **Debug**.

If it's important to keep it as **Information**, then you should put the error/failure messages (`LogProcessingErrorAction`, `LogResolveFailedAction`, `LogPreconditionFailedAction`) as **Warning** or higher.

**Comment by JimBobSquarePants** [comment, marked as answer]
The logging levels are copied from the levels used by ASP.NET in their static file handler.

**Reply by JimBobSquarePants** [reply]
P.S this should be in Q&A framed as a question
